mood-analysis.rb

```diff
@@ -28,5 +30,89 @@ def analyze_mood(words)
   "05/11 Yay, yay, yay! I'm having a awfuly great day."
 ]
 
-puts analyze_mood(text[0])
-puts analyze_mood(text[1])
+
+(0..text.count-1).map do |i|
+  "#{text[i].split(" ")[0]} #{analyze_mood(text[i])}"
+ [text[i].split(" ")[0], analyze_mood(text[i])]
+end
+
+
+
+def strip_punctuation(words)
+   words.gsub(/[!.#,]/, "")
+end
+
+def happy_days(text)
+count = 0
+happy = 0
+sad = 0
+neutral = 0
+(0..text.count-1).map do |i|
+  days =  [text[i].split(" ")[0], analyze_mood(text[i])]
+  count += 1
+  if days[1].include? "("
+  sad += 1
+  elsif days[1].include? "|"
+  neutral += 1
+  else days[1].include? ")"
+  happy += 1
```

Comment:
Ooooh! This is interesting. I hadn't thought to do it that way. It definitely looks like it works, but I think it would have made for more readable code if you said `days[1] == ":-("` etc.

The body's of the if's need to be indented here.